README.md

```diff
@@ -45,7 +45,7 @@ To join us in these efforts, please feel free to reach out, raise issues or read
 
 <hr>
 
-👋 **UPDATE - 02/09/2022**: As mentioned in our previous update, we are in the process of fully moving to JAX. In the next few weeks, we will release our first JAX system! Using a more modular and flexible build method, with many more develops to come soon. These will include more components and utilities, state-of-the-art system implementations, notebooks and tutorials as well as detailed benchmarking and performance guides. Please note that all TF2-based systems using the old build method will be deprecated in the near future. As part of this update, we have also restructured the readme and our documentation to reflect our move to JAX.
+👋 **UPDATE - 01/10/2022**: We have deprecated our TF2-based systems, to still use them please install [`v0.1.3`](https://github.com/instadeepai/Mava/releases/tag/0.1.3) of Mava (e.g. `pip install id-mava==0.1.3`). We will no longer be supporting these systems as we have moved to JAX-based systems.
```

Comment:
Can we keep this bit in the readme: "In the next few weeks, we will release our first JAX system! Using a more modular and flexible build method, with many more developments to come soon. These will include more components and utilities, state-of-the-art system implementations, notebooks and tutorials as well as detailed benchmarking and performance guides." Also, please see my comment on the timing of this PR below.

Reply:
Updated.